pkg/operator/ceph/csi/spec.go

```diff
@@ -307,3 +312,47 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 	}
 	return nil
 }
+
+// ValidateCSIVersion checks if the configured ceph-csi image is supported
+func ValidateCSIVersion(namespace string, clientset kubernetes.Interface, csiPluginImage string, rookImage string, serviceAccountName string) (bool, error) {
```

Comment:
Looks like a straight copy of  `detectCephVersion()`, any chance to make `detectCephVersion` more generic maybe and use it for both ceph and csi?